code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -230,7 +230,10 @@
 	amount_per_transfer_from_this = 10
 	flags = FPRINT  | OPENCONTAINER | NOREACT
 
-/obj/item/weapon/reagent_containers/glass/beaker/noreactlarge
+/obj/item/weapon/reagent_containers/glass/beaker/noreact/UsesBluespace()

```

Comment:
i'm the guy who coded it originally for tg, i ported it to bs12 and then you guys ported it

the intended downside is the low storage amount

i can understand actual bluespace beakers being nerfed but my original lore for the stasis beaker was "super-deep-freeze chemicals inside"